.travis.yml

```diff
@@ -7,6 +7,7 @@ python:
   - 3.5
   - 3.6
   - pypy-5.4.1
+  - pypy3.5-5.8.0
```

Comment:
We've reached the point where all Travis CI builds are running on Trusty, and so just a simple `pypy` and `pypy3` [get us the latest values](https://github.com/zopefoundation/zope.vocabularyregistry/pull/2). It might be good to use that? Hopefully Travis stays (more) up-to-date by default now.

Reply:
Thank you for your hint.